package.json

```diff
@@ -42,6 +42,7 @@
     "@types/redux-logger": "3.0.9",
     "@types/styled-components": "5.1.12",
     "@types/styled-system": "5.1.13",
+    "@yearn/web-lib": "^0.2.24",
```

Comment:
We probably should lock this version even though it's an internal package

Reply:
yes, lock

Reply:
Fixed with [5d07fe3](https://github.com/yearn/yearn-finance-v3/pull/622/commits/5d07fe342ecf54d9d45aefc2c116073f462ce185)